gradio/components.py

```diff
@@ -382,25 +382,43 @@ def deserialize(self, x):
         """
         return x
 
-    def change(self, fn: Callable, inputs: List[Component], outputs: List[Component]):
+    def change(
+        self,
+        fn: Callable,
+        inputs: List[Component],
+        outputs: List[Component],
+        status_tracker: Optional[StatusTracker] = None,
```

Comment:
A question here, why do we provide status_tracker to user?

Reply:
I think the reason is so that users can create a status tracker for arbitrary events in Blocks

@aliabid94 it would be great to see an example of this in one of the Blocks demos

Reply:
Sure I'll make another demo